src/app/pipes/timedisplay.ts

```diff
@@ -0,0 +1,29 @@
+import { PipeTransform, Pipe } from '@angular/core';
+
+
+@Pipe({name: 'timeDisplay'})
+export class TimeDisplayPipe implements PipeTransform {
+    transform(timeInSeconds: number): string {
```

Comment:
There's no available library function to calculate days/hours/minutes?

Reply:
Not that I could find sadly. We also wanted to have it show hours/minutes/seconds if it was less than a day, minutes and seconds if less than an hour, and then only seconds if less than a minute, so this seemed easier to just customize.